docs/guide/installation.md

```
@@ -0,0 +1,172 @@
+```
+
+React 项目将 `vue()` 换成 `viteReact()`（`@vitejs/plugin-react`），其余 `zelpis` 结构相同。
+
+入口文件需 `export default boot({ ... })`
+详见 [快速开始](/guide/quick-start/) 与 [示例项目](/examples/)。
+
+---
+
+## 推荐目录（商场模版项目）例子
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)